collagraph/collagraph.py

```diff
@@ -19,6 +20,8 @@
 
 logger = logging.getLogger(__name__)
 
+VIRTUAL_NODES = {"template", "slot"}
```

Comment:
The name is a little confusing since you also have a VNode class

Reply:
Ah, right :) Would `VIRTUAL_NODE_TAGS` or `VIRTUAL_NODE_TYPES` work for you?

Reply:
Yeap!

Reply:
👍 Changed it to `VIRTUAL_NODE_TYPES`, thanks!